fds/utils.py

```diff
@@ -134,12 +134,64 @@ def get_input_from_user(question: str, type: str = "input") -> str:
     :param type: The type of input
     :return: output from the user
     """
-    questions = [
-        {
-            'type': type,
-            'message': question,
-            'name': 'question'
-        }
-    ]
-    answers = PyInquirer.prompt(questions)
-    return answers['question']
+    get_input = input
+    if type == "password":
+        get_input = getpass.getpass
+    display_text_to_user = f"{question}:"
+    answer = get_input(display_text_to_user)
+    if answer is None:
+        return get_input_from_user(question, type)
+    return answer
+
+
+def get_expand_input_from_user(question: str, choices: List[Dict[str, str]], default: Union[str, bool, int]) -> str:
+    """
+    Get expand input from the user
+    :param question: Question to ask the user
+    :param choices: List of choices from user
+    :param default: Default selected value
+    :return: user selected value
+    """
+    # Only pick choice keys
+    choice_keys = list(map(lambda x: x["key"], choices))
+    # Add an extra 'h' for help
+    choice_keys.append("h")
+    choices_string = "".join(choice_keys)
+    default_choice = list(filter(lambda x: x['value'] == default, choices))[0]
+    default_key = default_choice["key"]
+    display_text_to_user = f"{question}   ({choices_string}) [{default_key}]"
+    input_value = input(display_text_to_user) or default_choice["key"]
+    while input_value == 'h' or input_value == 'help':
+        # User chose help then we should show the list again with full choices
+        detailed_choices = list(map(lambda x: f"{x['key']}) {x['name']}", choices))
```

Comment:
Just a matter of taste, but am I the only one who prefers the list comprehension syntax?
```suggestion
        detailed_choices = [f"{x['key']}) {x['name']}" for x in choices]
```